build-packages.sh

```diff
@@ -3,16 +3,16 @@
 usage()
 {
     echo "Builds the NuGet packages from the binaries that were built in the Build product binaries step."
-    echo "Usage: build-packages -BuildArch -BuildType [-portable]"
+    echo "Usage: build-packages -BuildArch -BuildType [-nonportable]"
     echo "BuildArch can be x64, x86, arm, arm64 (default is x64)"
     echo "BuildType can be release, checked, debug (default is debug)"
-    echo "-portable - build for Portable Distribution"
+    echo "-nonportable - build for Distro-specific RID"
```

Comment:
I considered changing this to non-portable in corefx but instead I just kept the portable flag and made our config.json always pass it unless you pass portable=false.  I'm not sure if you want to follow a similar pattern here as well.